src/Utils/logger.ts

```diff
@@ -0,0 +1,28 @@
+import { head } from "lodash"
+import { isErrorInfo, sendErrorToService } from "Utils/errors"
+
+export const shouldCaptureError = (environment: string) =>
+  environment === "staging" || environment === "production"
+
+export default function createLogger(namespace = "reaction") {
+  const formattedNamespace = `${namespace} |`
+
+  return {
+    log: (...messages) => {
+      console.log(formattedNamespace, ...messages, "\n")
+    },
+    warn: (...warnings) => {
+      console.warn(formattedNamespace, ...warnings, "\n")
+    },
+    error: (...errors) => {
+      const error = head(errors.filter(e => e instanceof Error))
+      const errorInfo = head(errors.filter(e => isErrorInfo(e)))
```

Comment:
These `head(….filter(…))` combinations can just be replaced by the builtin `Array.prototype.find()` method, no? Doesn’t require an external dependency (lodash) and it’s a little faster because it will stop iteration after finding the first match.

Reply:
Additionally, when all your function does is invoke another function with the same parameters your function received, you can simplify it somewhat (and get rid of another closure) by just passing the target function. `errors.find(isErrorInfo)`

Reply:
Good point, I'll change that.